### PR TITLE
Project Execution - D1 & E1

### PR DESCRIPTION
**Pull request by McDoxa**
Implementing Projects D1 and E1

### CODE REVIEW

**Comment by McDoxa** [comment]
Apologies. I have pushed again (ef883b6). Please check. If it is still empty, I might have to do another pull request. Thank you Sir.

File: project-javascript-d1-functions/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        }
+        let dayName = showDayName(dayOfWeek); // calling the function showDayName
+
+        //5. A function that converts a number to string.
+        let userInput = 4;
+        function convert2String(userInput){
+            if (userInput < 10){
+                return `0${userInput}`;
+            }
+            return userInput;
```
Comment (basir) [inline project-javascript-d1-functions/Oluwaseyi.html]: It does not conver number >= 10 to string. Use:
`return userInput + '';`

File: project-javascript-d1-functions/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        console.log(convert); // logging the output of the function to the console.
+
+        //6. A function that formats date.
+        function formatDate(year, month, day){
+             formatDay = convert2String(day);
+             formatMonth = convert2String(month);
+             formatYear = convert2String(year);
+             return `${formatYear}/${formatMonth}/${formatDay}`;
+        }
+        todaysDate = formatDate(2008,2,8); // calling the function formatDate
```
Comment (basir) [inline project-javascript-d1-functions/Oluwaseyi.html]: Use const to define a variable.

File: project-javascript-d1-functions/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        console.log(todaysDate); //logging to the console.
+
+        //7. Setting default parameters
+        function formatDateChristmas(year = 2020, month = 01, day=01){
+            return formatDate(year,month,day);
+        }
+        xmas = formatDateChristmas() // calling function formatDateChristmas
```
Comment (basir) [inline project-javascript-d1-functions/Oluwaseyi.html]: same as above

File: project-javascript-d1-functions/Oluwaseyi.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        console.log(xmas) // logging the output to the console
+
+        //8. Calling a function as a parameter in another function.
+        function nextYear(year){
+            return year + 1;
+        }
+        abc = formatDate(nextYear(2006),3,7);
```
Comment (basir) [inline project-javascript-d1-functions/Oluwaseyi.html]: same as above

File: project-javascript-e1-scope/Oluwaseyi.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (basir) [inline project-javascript-e1-scope/Oluwaseyi.html]: One project at a time. This pull request is for only Project D1. Delete this file and do it in another pull request.
First apply my changes to accept this one.

**Comment by McDoxa** [comment]
Changes have been effected and Project E1 deleted. Thank you.

**Comment by basir** [comment]
go for next project
